Whiskey/Tasks/ProGetUniversalPackage.ps1

```diff
@@ -129,12 +127,10 @@ function New-WhiskeyProGetUniversalPackage
     function Copy-ToPackage
     {
         param(
-            [Parameter(Mandatory=$true)]
-            [object[]]
-            $Path,
+            [Parameter(Mandatory)]
+            [object[]]$Path,

```

Comment:
Should use the same case as the fully qualified type name `[System.Object]` as no `[object]` type accelerator exists:
```
❯ ([PSObject].Assembly.GetType('System.Management.Automation.TypeAccelerators')::Get).GetEnumerator() | Where-Object { $_.Value.Name -eq 'Object' }

❯
```